english/die/youre-dead.md

```diff
@@ -0,0 +1,11 @@
+Oops, you died.
+
+You appear to be in some sort of afterlife...
+
+There is nothing but darkness and thick fog for miles around, except for one strange old man with a beard longer than an elephant's trunk.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.